spec/controllers/admin/product_import_controller_spec.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe Admin::ProductImportController, type: :controller do
+  describe 'tmp_file_path_regex' do
```

Comment:
Wouldn't it be better to test `#validate_file_path` instead?

Reply:
This method is private., is it ok if I use `send` method to reach it?